constph.py

```diff
@@ -55,9 +55,17 @@
 import simtk.unit as units
 import pymbar
 
+def strip_unit(quant):
```

Comment:
For added safety, I would recommend changing the call signature to

```Python
def strip_unit(quant, unit):
```

Where `unit` is the target unit.  That way, you can properly throw an exception when the input units are incompatible with what's expected.  This is a potential cause of bugs.  For instance, one of my friends once instantiated a LangevinIntegrator as such:

```
integrator = LangevinIntegrator(1/femtosecond, 300*kelvin)
```

only to have the simulation immediately segfault because the time step was expanded to 2000/picosecond instead of 0.002 picosecond (and the unit implicitly dropped).  The only reason we can't use this approach in OpenMM is because the code is embedded in the SWIG/C++ translation layer, and it's a real challenge to accommodate everywhere that it's needed.